contribs/src/main/java/com/netflix/conductor/contribs/http/GenericHttpTask.java

```diff
@@ -145,6 +146,9 @@ HttpResponse httpCall(Input input, Task task, Workflow workflow, WorkflowExecuto
 		if (headers.containsKey(HttpHeaders.AUTHORIZATION)) {
 			headers.put(HttpHeaders.AUTHORIZATION, "xxxxxxxxxxxxxxxxxxx");
 		}
+		if (headers.containsKey("Authorization-Context")) {
```

Comment:
Does it make sense to define `HttpHeaders.AUTHORIZATION_CONTEXT` instead of using a literal?

Reply:
@hueys  I agree, @sangeetanadgir  just like we use HttpHeaders.AUTHORIZATION we need to use same way instead of hard coded value.

Reply:
@heajuenhwang @hueys  HttpHeaders class is from library not a customized class so it may not be possible to use HttpHeaders.AUTHORIZATION_CONTEXT .We may need to create a custom class for configuration and read it from there. Let me know if you have any other ideas on this.

Reply:
If it can't be added to HttpHeaders than I think it should either be added somewhere else that makes sense, or in a new file. The string literal is used multiple places and using a constant instead to avoid typos and get compile time checks would be a good idea.

Reply:
@hueys  @heajuenhwang Reading auth-context header value from common class is implemented.